source/GlassView.Core.Test/Serialization/SerializationTest.cs

```diff
@@ -0,0 +1,63 @@
+using Atmoos.GlassView.Core.Models;
+
+namespace Atmoos.GlassView.Core.Test;
+
+public class SerializationTest
+{
+    private static readonly String testFile = Path.Combine("Resources", "TestBenchmark.json");
+
+    [Fact]
+    public void SerializationIsPossible()
+    {
+        String expected = File.ReadAllText(testFile);
+        BenchmarkSummary benchmarks = expected.Deserialize<BenchmarkSummary>();
+
+        String actual = benchmarks.Serialize();
+
+        Assert.Equal(expected, actual, ignoreAllWhiteSpace: true, ignoreLineEndingDifferences: true);
```

Comment:
the whitespace & LineEndings are for Linux vs Win, correct?

Reply:
Yes, you are correct. Also the `JsonFormatting.Indented` parameter can change how serialization formats the output. For this test that is all irrelevanr and we only (shoud) care for the "key & value" nodes of the exported json.